VulkanHppGenerator.cpp

```diff
@@ -405,6 +405,7 @@ void VulkanHppGenerator::generateVulkanRAIIHppFile() const
 #  define VULKAN_RAII_HPP
 
 #include <memory>
+#include <utility> // std::exchange
```

Comment:
Please include `utility` only if `( 14 <= VULKAN_HPP_CPP_VERSION )`.

Reply:
both `std::exchange` and `std::forward` (https://en.cppreference.com/w/cpp/utility/forward) require `<utility>`